LavaFlow.cdc

```diff
@@ -198,6 +199,23 @@ pub contract LavaFlow {
         }
     }
 
+    pub resource PlayerMinter {
```

Comment:
Can you add a comment?

Reply:
What should go in a comment?

Reply:
You can look at the other comments. Intent and light description is usually enough.

I follow Go’s style where everything that is a public API gets a comment.